db-client/src/test/component/db.test.ts

```diff
@@ -43,6 +43,8 @@ test.beforeEach(async (t) => {
     `http://localhost:${port}/v1`,
     {
       appId: 'testing',
+      appEnv: 'default',
+      collection: 'default',
```

Comment:
Might be confusing that both env and collection are default...

Reply:
Hmm.. that's what we use ATM in the deployed version.